cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -1211,8 +1213,9 @@ func getDisks() ([]*provision.Disk, error) {
 		}
 
 		disks = append(disks, &provision.Disk{
-			// add 1 MB to make extra room for GPT and alignment
-			Size:            diskSize + 2*1024*1024,
+			// add 2 MB per partition to make extra room for GPT and alignment
+			// align to 4MB due to QEMU limitations
+			Size:            (diskSize + 2*1024*1024*uint64(len(diskPartitions)+1) + QEMUAlignment - 1) / QEMUAlignment * QEMUAlignment,
```

Comment:
Maybe extract this variable and create it step by step like this for clarity:
```go
const GPTExtraSpace = 2*1024*1024
size := diskSize + GPTExtraSpace*uint64(1+len(diskPartitions))
size = (size + QEMUAlignment - 1) / QEMUAlignment * QEMUAlignment
```

Reply:
but I can do the change you proposed as well

Reply:
Okay, then that'd be a different name, yet still not a number

Reply:
I refactored this completely moving the QEMU alignment into QEMU provisioner applying to any disk types, while GPT gap is still here